sample-apps/react/react-dogfood/sentry.client.config.ts

```diff
@@ -31,6 +31,7 @@ if (window.location.href.includes('no-trace')) {
         maskAllText: true,
         blockAllMedia: true,
       }),
+      Sentry.captureConsoleIntegration(),
```

Comment:
@szuperaz, I'm curious if this renders our `customSentryLogger` obsolete.

Reply:
Based on your description, yes. We just have to provide `console.log` as the logger function for the video client

Reply:
I'll keep everything as so we have an example on how this can be achieved. In case this creates some conflicts, we can remove/adjust the sentry logger.